bigtable/tests/unit/test_row_data.py

```diff
@@ -678,7 +681,7 @@ class _PartialCellData(object):
 
     row_key = ''
     family_name = u''
-    qualifier = b''
+    qualifier = None
```

Comment:
This is to match [`PartialCellData`][1], but that class has no defaults (i.e. `None` isn't the default).

[1]: https://github.com/GoogleCloudPlatform/google-cloud-python/blob/6dbb16c02ba4cdf4624a23b88167465133cb45f1/bigtable/google/cloud/bigtable/row_data.py#L95-L102